frontend/apps/reader/modules/readerstyletweak.lua

```diff
@@ -427,12 +439,23 @@ function ReaderStyleTweak:onSaveSettings()
     end
     self.ui.doc_settings:saveSetting("style_tweaks", util.tableSize(self.doc_tweaks) > 0 and self.doc_tweaks or nil)
     G_reader_settings:saveSetting("style_tweaks", self.global_tweaks)
+    G_reader_settings:saveSetting("style_tweaks_registered", self.registered_tweaks)
     self.ui.doc_settings:saveSetting("book_style_tweak", self.book_style_tweak)
     self.ui.doc_settings:saveSetting("book_style_tweak_enabled", self.book_style_tweak_enabled)
     self.ui.doc_settings:saveSetting("book_style_tweak_last_edit_pos", self.book_style_tweak_last_edit_pos)
 end
 
+local function dispatcherRegisterStyleTweak(tweak_id, tweak_title)
+    Dispatcher:registerAction("style_tweak_"..tweak_id,
+        {category="none", event="ToggleStyleTweak", arg=tweak_id, title=T(_("Toggle style tweak: %1"), tweak_title), rolling=true})
+end
+
+local function dispatcherRemoveStyleTweak(tweak_id)
```

Comment:
Wording only again: do we use Register/Remove elsewhere ? I would expect Register/Unregister.

Reply:
https://github.com/koreader/koreader/blob/486d7071c7f08dca6b2d247c1d67dae1a615b123/plugins/profiles.koplugin/main.lua#L57-L64

Reply:
Didn't notice that at the time (or I was less interested by the profile plugin than by this styletweak module :).
I'd prefer to read all (these 2 only then?) as register/unregister - but up to you.

Reply:
Done